tox.ini

```diff
@@ -0,0 +1,68 @@
+# Tox (http://tox.testrun.org/) is a tool for running tests
+# in multiple virtualenvs. This configuration file will run the
+# test suite on all supported python versions. To use it, "pip install tox"
+# and then run "tox" from this directory.
+
+[tox]
+envlist = py27,py36,flake8
+skip_missing_interpreters = False
+
+[travis]
+python =
+  3.6: py36, flake8
+
+[flake8]
+max-line-length = 120
+exclude =
+    build/
+    .git
+    __pycache__
+    examples/
+    *pb2.py
+    .tox
+    tests/data/
+    test/resources
+    venv/
+    sagemaker-tensorflow-extensions
+max-complexity = 10
+ignore =
+    FI10,
+    FI12,
+    FI13,
+    FI14,
+    FI15,
+    FI16,
+    FI17,
+    FI50,
+    FI51,
+    FI52,
+    FI53,
+    FI54,
+    FI55,
+    FI56,
+    FI57,
+    E722
+
+require-code = True
+
+[testenv]
+# TEAMCITY_VERSION environment variable exists during build on Teamcity. teamcity-messages uses it in order to enable
+# reporting to TeamCity.
+passenv =
+    TEAMCITY_VERSION
+    AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY
+    AWS_SESSION_TOKEN
+# {posargs} can be passed in by additional arguments specified when invoking tox.
+# Can be used to specify which tests to run, e.g.: tox -- -s
+commands =
+    coverage run -m pytest {posargs}
+    {env:IGNORE_COVERAGE:} coverage report --fail-under=90 --include *sagemaker_tensorflow_container* --omit */tensorflow/tensorflow_serving/*
```

Comment:
You can separate code coverage between python 2 and python 3 following this example here https://github.com/aws/sagemaker-containers/blob/master/.coveragerc_py27

https://github.com/aws/sagemaker-containers/blob/master/tox.ini#L32

This file here configures codecov https://github.com/aws/sagemaker-containers/blob/master/codecov.yml